### DISCUSSION TITLE
Proud users of LNPopupController

### DISCUSSION THREAD

**Discussion by RajChanchal** [LeoNatan/LNPopupController#454] [Show and tell]
#### Description
Kindly provide some way to allow us to let you know that we are proud users of LNPopupController. 
A showcase of apps using LNPopupController would be great.

**Comment by LeoNatan** [comment]
I would love to have that, but not sure what the proper avenue is.

**Comment by RajChanchal** [comment]
Don't know myself, but I have seen repository owners providing a list of apps. Please do include an app I have worked on using your lib: [Mehad Hamad](https://itunes.apple.com/bj/app/mehad-hamad-myhd-hmd/id1014431425?mt=8)

**Comment by atulkhatri** [comment]
I'm a proud user of this repository as well. We have integrated this in our app [VuLiv Player](https://itunes.apple.com/in/app/vuliv/id1042979561). You may include it in your list.

Thanks

**Comment by MaticConradi** [comment]
[Helios - Magic hour calculator](https://itunes.apple.com/us/app/helios-golden-hour-blue-hour-calculator/id1260649184?mt=8)

**Comment by funkenstrahlen** [comment]
Will be using it soon: https://podlive.io/en

**Comment by noisy62** [comment]
Thanks for the excellent work! My App [OctaMusic](https://itunes.apple.com/us/app/OctaMusic/id1326625098?mt=8) is using it.
